### ISSUE TITLE
Missing projects for NUCLEO-G0B1RE

### ISSUE THREAD

**Issue by LongDirtyAnimAlf** [STMicroelectronics/x-cube-tcpp#6]
Hello again,

The datasheet about the X-NUCLEO-DRP1M1 states that it is suitable for the NUCLEO-G0B1RE.
As should be this library ((X-CUBE-TCPP).
![drp1m1](https://user-images.githubusercontent.com/10783972/151806060-5c4c095e-f4dd-497a-9cba-b86006eca215.png)
I have invested a lot of money and time into a project using parts by STM that are based on this combination.
Could you please elaborate on your dismissal of the G0B1RE ?

**Comment by GCASTM** [comment]
Hello, 
First, I confirm that X-NUCLEO-DRP1M1 expansion board is suitable for the NUCLEO-G0B1RE, i.e. it is possible to use the expansion shield on top of NUCLEO-G0B1RE and build a DRP application.
Secondly, our X-CUBE-TCPP expansion package aims to provide examples of applications for different HW configurations. The goal is not to provide all examples for all boards. Switching an existing example from a G071 to a G0B1 is feasible, even if not provided in the X-CUBE package.

My understanding was that you were requesting a pull request, for adding your G0B1 project in X-Cube-TCPP, so I guess it should be working fine. If it is not the case, and if you encounter some issues in doing this switch, we are ready to support you. Regards

**Comment by LongDirtyAnimAlf** [comment]
Thanks for explaining.
However, can you please explain this reaction ?
https://github.com/STMicroelectronics/x-cube-tcpp/pull/5#issuecomment-1023403869

**Comment by LongDirtyAnimAlf** [comment]
Bump.
As I explained, I have invested time and money into the G0B1 for this project.
Your colleague deems this invalid.
What is going on ?

**Comment by GCASTM** [comment]
Let me rephrase my previous comment.
ST delivers some "examples" of running applications in this X-CUBE. These are examples running on a specific HW setup and could be used as basis for developers using other HW. But ST does not intend to provide the same examples for all possible HW configurations.
So, starting from the G081 example, you should be able to make the G0B1 flavor running similarly, with only changing target device. Isn't it the case ? Do you still have problems ?

By the way, the comment in the UM will be corrected to make this clearer.
Thanks for pointing this out.

**Comment by HFISTM** [comment]
Hello,
UM2891 documentation has been updated, thank you for reporting this inconsistency.
I'm closing this issue now.
Best regards
